ambuda/views/proofing/page.py

```diff
@@ -110,43 +111,53 @@ def add_revision(
     return new_version
 
 
+def _split_graphemes(s: str) -> str:
+    """Splits the given string into graphemes and returns
+    a list of those graphemes. Taking a slice of the list
+    returns a string (rather than a list) so that the list
+    can be passed into difflib.SequenceMatcher properly."""
+
+    class CustomList(list):
```

Comment:
is there any benefit to creating this per call? If not, let's move it up one level and give it a better name (maybe `GraphemeList`?).